e2e/pages/StageOne/PasswordPolicyPage.ts

```diff
@@ -22,4 +58,53 @@ export default class PasswordPolicyPage {
       }
     );
   }
+
+  // --- ON LOAD METHODS --- //
+
+  async assertButtonsVisible() {
+    assertComponentsVisible([this.backButton, this.continueButton]);
+  }
+
+  async assertTextVisible() {
+    assertComponentsVisible([
+      this.passwordPolicyHeading,
+      this.passwordText1,
+      this.passwordText2,
+      this.passwordText3,
+    ]);
+  }
+
+  async assertFormVisible() {
+    assertComponentsVisible([
+      this.createPasswordInput,
+      this.confirmPasswordInput,
+      this.createPasswordLabel,
+      this.confirmPasswordLabel,
+    ]);
+  }
+
+  async assertErrorsHidden() {
```

Comment:
See comment about same thing on name page.
Nothing wrong with this, but since you've made that clever util method for visible, may as well do the same with hidden